graphql-server/pkg/dataloaders/block.go

```diff
@@ -27,4 +41,37 @@ func NewBlockDataloader(blockQuery queries.IBlockQuery) BlockDataloader {
 		MaxBatch: DefaultMaxBatch,
 		Wait:     DefaultWait,
 	})
+
+	blockByHeightLoader := godataloader.NewDataLoader(godataloader.DataLoaderConfig[int64, *models.Block]{
+		Fetch: func(heights []int64) ([]*models.Block, []error) {
+			blocks, err := blockQuery.QueryBlocksByHeights(heights)
```

Comment:
do we usually map to ordered results at query or data loader layer in this project?

Reply:
(just stick to usual pattern at this point, but seems weird to put logic for fitting results to data loader style at query level)

Reply:
I think it's alright since bun also has provided CTE way to handle similar cases in query level (except it doesn't work)

so it's not that we ordered the result to fit dataloader but instead dataloader just so happens to use ordered results to function :hehe: (jk)